packages/libs/error-stack/tests/snapshots/doc/fmt__charset_ascii.snap

```diff
@@ -0,0 +1,9 @@
+<b>invalid input parameter</b>
+<b>|- </b><i>src/fmt/charset.rs:21:5</i>
```

Comment:
I'm not sure but do we want to add the `"at "` here as well? If so, we could also remove the whitespace between `-` and `at` again I guess.

Reply:
🤔 I am unsure. We could if that is the case, would it be "at <color>LOCATION</color>" where color is either color or emphasis or another way?